pkgs/top-level/php-packages.nix

```diff
@@ -561,8 +537,6 @@ lib.makeScope pkgs.newScope (self: with self; {
         {
           name = "xmlrpc";
           buildInputs = [ libxml2 libiconv ];
-          # xmlrpc was unbundled in 8.0 https://php.watch/versions/8.0/xmlrpc
-          enable = lib.versionOlder php.version "8.0";
```

Comment:
This is actually the opposite, we need to remove the xmlrpc extension.

Reply:
Ah, right, pushed a fixup :slightly_smiling_face: